iotlabcli/rest.py

```diff
@@ -330,6 +331,8 @@ def method(self, url, method='get',  # pylint:disable=too-many-arguments
                             json=json, files=files)
         if requests.codes.ok == req.status_code:
             return req.content if raw else req.json()
+        if requests.codes.no_content == req.status_code:
```

Comment:
Hi 
I suggest for this case to return a value instead of None. Maybe we should use an empty string ? At the end it will use in print_result method

Reply:
There is a 
```
    if result is None:
        return
```
in print_result, so nothing is printed, just like if an empty string was used. Example in CLI when deleting a profile (which returns 204 on success)
```
(cli-tools) matthieu@zurkus:~/iot-lab-dev/parts/cli-tools (new_api)$ iotlab-profile del -n rssi
(cli-tools) matthieu@zurkus:~/iot-lab-dev/parts/cli-tools (new_api)$
```
`204 No Content` fits better with None than `` ("some content", just an empty string) In my opinion